File: app/src/main/java/com/example/networkaplication/persistance/repository/database/DatabaseRepository.java
package com.example.networkaplication.persistance.repository.database;

import com.example.networkaplication.persistance.repository.database.dao.DetailsDao;
import com.example.networkaplication.persistance.repository.database.dao.MovieQueryDao;

public interface DatabaseRepository {

    MovieQueryDao movieQueryDao ();

    DetailsDao detailsDao();
}

File: app/src/main/java/com/example/networkaplication/persistance/repository/database/DatabaseRepository.kt
package com.example.networkaplication.persistance.repository.database

import com.example.networkaplication.persistance.repository.database.dao.DetailsDao
import com.example.networkaplication.persistance.repository.database.dao.MovieQueryDao

interface DatabaseRepository {

    fun movieQueryDao(): MovieQueryDao

    fun detailsDao(): DetailsDao
}
